ct-app/core/api/response_objects.py

```
@@ -51,7 +51,7 @@ def is_null(self):
 
     @property
     def as_dict(self) -> dict:
-        return {key: getattr(self, key) for key in self.keys.keys()}
+        return {key: str(getattr(self, key)) for key in self.keys.keys()}
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)